### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sduzh**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: 'type/bug'
+
+---
+
+### Steps to reproduce the behavior (Required)
+1. Create table '...'
```
Comment (Linkerist) [inline .github/ISSUE_TEMPLATE/bug-report.md:9]: All uppercase or all lowercase.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+2. Insert into '....'
+3. Select  '....'
```
Comment (Linkerist) [inline .github/ISSUE_TEMPLATE/bug-report.md:11]: All uppercase or all lowercase.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
```
Comment (Linkerist) [inline .github/ISSUE_TEMPLATE/feature-request.md:2]: No need to specifically point out that it is a request, feature/bug/or something else are belong to request,

**Review by Linkerist — APPROVED** [review]

**Comment by sduzh** [comment]
@kangkaisen PTAL

**Review by kangkaisen — APPROVED** [review]

**Comment by kangkaisen** [comment]
Nice job!

**Pull request merged**